src/components/SignerPanel/SignerPanel.js

```diff
@@ -174,76 +174,94 @@ class SignerPanel extends React.Component {
-                          walletNetworkType={walletNetwork}
-                          walletProviderId={walletProviderId}
-                        />
-                      </Screen>
-                    </ScreenWrapper>
-                  )
-                : ({ enterProgress }) => (
-                    <ScreenWrapper
-                      style={{
-                        transform: `
+                        }}
+                      >
+                        <Screen>
+                          <ConfirmTransaction
+                            direct={directPath}
+                            hasAccount={Boolean(account)}
+                            hasWeb3={Boolean(walletWeb3)}
+                            intent={intent}
+                            locator={locator}
+                            networkType={network.type}
+                            onClose={this.handleSignerClose}
+                            onRequestEnable={onRequestEnable}
+                            onSign={this.handleSign}
+                            paths={actionPaths}
+                            pretransaction={pretransaction}
+                            signingEnabled={status === STATUS_CONFIRMING}
+                            walletNetworkType={walletNetwork}
+                            walletProviderId={walletProviderId}
+                          />
+                        </Screen>
+                      </ScreenWrapper>
+                    )
+                  : ({ enterProgress }) => (
+                      <ScreenWrapper
+                        style={{
+                          transform: `
                           translate3d(${100 * (1 - enterProgress)}%, 0, 0)
                         `,
-                      }}
-                    >
-                      <Screen>
-                        <SigningStatus
-                          status={status}
-                          signError={signError}
-                          onClose={this.handleSignerClose}
-                          walletProviderId={walletProviderId}
-                        />
-                      </Screen>
-                    </ScreenWrapper>
-                  )
-            }
-          </Transition>
-        </Main>
-      </SidePanel>
+                        }}
+                      >
+                        <Screen>
+                          <SigningStatus
+                            status={status}
+                            signError={signError}
+                            onClose={this.handleSignerClose}
+                            walletProviderId={walletProviderId}
+                          />
+                        </Screen>
+                      </ScreenWrapper>
+                    )
+              }
+            </Transition>
+          </Main>
+        </SidePanel>
+      </ResponsiveSidePanel>
     )
   }
 }
 
+const ResponsiveSidePanel = styled.div`
+  ${isSmallScreen() &&
```

Comment:
Shall we use a media query here? Using these styles as a default, and `breakpoint('medium')` to adapt the styles to larger sizes. Otherwise the component would not adapt to screen size updates (e.g. changing the orientation of a device, resizing a window).